src/utils/mod.ts

```diff
@@ -1,21 +1,4 @@
-import { semverGreaterThanOrEquals, semverParse } from "../../deps.ts";
-
 export { parseEntrypoint } from "./entrypoint.ts";
 export { API, APIError } from "./api.ts";
 export { convertPatternToRegExp, walk } from "./walk.ts";
-export { fromFileUrl, resolve } from "../../deps.ts";
-
-// deno-lint-ignore no-explicit-any
-export function isTerminal(stream: any) {
-  if (
-    semverGreaterThanOrEquals(
-      semverParse(Deno.version.deno),
-      semverParse("1.40.0"),
-    )
-  ) {
-    return stream.isTerminal();
-  } else {
-    // deno-lint-ignore no-deprecated-deno-api
-    return Deno.isatty(stream.rid);
-  }
-}
```

Comment:
For reviewers: `isTerminal` method on various stream types is available on 1.40.0+ and our minimum required Deno version is now 1.46.0. We don't have to polyfill anymore.